src/components/DynamicFormContainer/DataFileButton/DataFileButton.tsx

```diff
@@ -113,37 +113,35 @@ export const DataFileButton: FunctionComponent<DataFileButton> = ({ onDataFile,
           <FormErrorBanner formErrorTitle="Uploaded data file format has errors." formError={dataFileError} />
         </div>
       )}
-      <div className="p-8 font-light" style={{ backgroundColor: "#FFECA7" }}>
-        <p className="text-sm font-hairline mb-4">{text.header}</p>
+      <div className="p-8 rounded-xl border-dashed border-2" style={{ backgroundColor: "#FFF7E2" }}>
```

Comment:
if can reuse closest bg color will be great. maybe `bg-gray-50`? not 1 to 1 but saves the trouble of maintainence...

Reply:
the original color is yellow leh not gray hahaha. ask jerry eh cos he add this color not in the styleguide

Reply:
okok then nvm lets inline this